hermes-common/src/main/java/pl/allegro/tech/hermes/common/http/MessageMetadataHeaders.java

```diff
@@ -3,7 +3,8 @@
 public enum MessageMetadataHeaders {
 
     MESSAGE_ID("Hermes-Message-Id"),
-    TOPIC_NAME("Hermes-Topic-Name");
+    TOPIC_NAME("Hermes-Topic-Name"),
+    TRACE_ID("Hermes-Trace-Id");
```

Comment:
since it uses some custom header, we should also change hermes-client to easily pass it on in request